source/help/getting-started/searching.rst

```diff
@@ -26,6 +26,17 @@ posts in specific channels.
 -  For example: Searching ``Mattermost in:town-square`` only returns
    messages in Town Square that contain ``Mattermost``.
 
+Before:, After: and On:
+^^^^^^^^^^^^^^^^^^^^^^
+
+Use ``before:`` to find posts before a specified date and ``after:`` to find posts after a specified date. Use both ``before:`` and ``after:`` together to search in a specified date range.  Use ``on:`` to find posts from a specific date.  You can use the date picker to select a date or you can type it in YYYY-MM-DD format.
+
```

Comment:
Propose adding a screenshot of the date selector UI here? Since this is an end user guide images are helpful

Reply:
Note the styling is updated in the next RC so maybe wait till that happens to take the screenshot

Reply:
Thanks Amy!